Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs

```diff
@@ -300,27 +300,27 @@ public bool CanInsert(EntityUid uid, EntityUid usedUid, EntityUid? user, ItemSlo
         ///     Tries to insert item into a specific slot.
         /// </summary>
         /// <returns>False if failed to insert item</returns>
-        public bool TryInsert(EntityUid uid, string id, EntityUid item, EntityUid? user, ItemSlotsComponent? itemSlots = null, bool excludeUserAudio = false)
+        public bool TryInsert(EntityUid uid, string id, EntityUid item, EntityUid? user, ItemSlotsComponent? itemSlots = null, bool excludeUserAudio = true)
         {
             if (!Resolve(uid, ref itemSlots))
                 return false;
 
             if (!itemSlots.Slots.TryGetValue(id, out var slot))
                 return false;
 
-            return TryInsert(uid, slot, item, user, excludeUserAudio: excludeUserAudio);
+            return TryInsert(uid, slot, item, user, excludeUserAudio: default);
```

Comment:
You must forward the `excludeUserAudio` as was passed in to this method, cannot override it to `default` (which is false, for boolean).

Also in other places.